performance/performance_tuning.rst

```diff
@@ -411,9 +411,9 @@ From a throughput standpoint, you can expect data ingest ratios of 250K - 2M row
   * Go with a relatively small shard count, 32 should suffice but you could benchmark with 64, too.
   * Ingest data for a suitable amount of time (say 2, 4, 8, 24 hrs). Longer tests are more representative of a production setup.
 
-Masterless Citus (50k/s-500k/s)
-===============================
+Citus MX (50k/s-500k/s)
+=======================
 
-Masterless Citus (Citus MX) builds on the Citus extension. It gives you the ability to query and write to distributed tables from any node, which allows you to horizontally scale out your write-throughput using PostgreSQL. It also removes the need to interact with a primary node in a Citus cluster for data ingest or queries.
+Citus MX builds on the Citus extension. It gives you the ability to query and write to distributed tables from any node, which allows you to horizontally scale out your write-throughput using PostgreSQL. It also removes the need to interact with a primary node in a Citus cluster for data ingest or queries.
 
 Citus MX is currently available in private beta on Citus Cloud. For more information see :ref:`mx`.
```

Comment:
We probably want to change this to: Citus MX is available in our Enterprise Edition and Citus Cloud. For more information ..."

Reply:
D'oh. Just did a search to find all references to "beta" and fixed that.